### PR TITLE
add example using cookie to store JWT token

### PR DESCRIPTION
**Pull request by mandaputtra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,6 +142,74 @@ fastify.register(jwt, {
...
+  .register(require('fastify-redis'), {client: redis})
+  .register(require('fastify-cookie'))
+  .register(require('fastify-caching'), {cache: abcache})
+  .register(require('fastify-server-session'), {
+    secretKey: 'your secret key',
+    sessionMaxAge: 900000 // 15 minutes in milliseconds
+  })
+
+fastify.get('/cookies', async (request, reply) => {
+  const token = this.jwt.sign({
```
Comment (Eomm) [inline README.md]: Could you try and format with `standard` this example?
Because with this setup I don't think it works (`this` here is an empty object)

Comment (mandaputtra) [reply README.md]: Oh okay. forgot this.

File: README.md [hunk 2]
```diff
@@ -213,6 +212,78 @@ fastify.listen(3000, err => {
...
+  },
+  sign: { algorithm: 'ES256' }
+})
+
+fastify
+  .register(require('fastify-redis'), { client: redis })
+  .register(require('fastify-cookie'))
+  .register(require('fastify-caching'), { cache: abcache })
+  .register(require('fastify-server-session'), {
+    secretKey: 'your secret key',
```
Comment (Eomm) [inline README.md]:
```suggestion
    secretKey: 'some-secret-password-at-least-32-characters-long',
```

File: README.md [hunk 3]
```diff
@@ -213,6 +212,78 @@ fastify.listen(3000, err => {
...
+You can use [crsf](https://www.npmjs.com/package/csrf) or other library that may suit your need.
+
+```js
+const { readFileSync } = require('fs')
+const path = require('path')
+const fastify = require('fastify')()
+const jwt = require('fastify-jwt')
+const Redis = require('ioredis')
+
+const redis = new Redis()
```
Comment (Eomm) [inline README.md]:
```suggestion
// docker run -p 6379:6379 --name redis-test redis
const redis = new Redis({ port: 6379, host: '127.0.0.1' })
```

Comment (mandaputtra) [reply README.md]: actualy, just `new Redis()` would be set to default port but this more informative.

**Pull request merged**